packages/xrpl/src/models/ledger/PayChannel.ts

```diff
@@ -24,17 +24,17 @@ export default interface PayChannel extends BaseLedgerEntry {
    */
   Destination: string
   /**
-   * Total XRP, in drops, that has been allocated to this channel. This
-   * includes XRP that has been paid to the destination address. This is
-   * initially set by the transaction that created the channel and can be
-   * increased if the source address sends a PaymentChannelFund transaction.
+   * Total amount that has been allocated to this channel. This includes amounts
+   * that have been paid to the destination address. This is initially set by the
+   * transaction that created the channel and can be increased if the source
+   * address sends a PaymentChannelFund transaction.
    */
   Amount: string
   /**
-   * Total XRP, in drops, already paid out by the channel. The difference
-   * between this value and the Amount field is how much XRP can still be paid
-   * to the destination address with PaymentChannelClaim transactions. If the
-   * channel closes, the remaining difference is returned to the source address.
+   * Total amount already paid out by the channel. The difference between this value
+   * and the Amount field is how much can still be paid to the destination address
+   * with PaymentChannelClaim transactions. If the channel closes, the remaining
+   * difference is returned to the source address.
    */
   Balance: string
```

Comment:
Don't these types need to change?

Reply:
Unresolving this because my understanding is these now need to be `Amount`s

Reply:
Does this not need to be `Balance: Amount` and `Amount: Amount`?

Reply:
Wow. Apologies and thank you.